pom.xml

```diff
@@ -87,7 +87,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>github-api</artifactId>
-            <version>1.71</version>
+            <version>1.86</version>
```

Comment:
I think that's a 2.60 dependency as well, but I'll revert and see if I can get a build working without this.